terraform/gcp/projects/pilot-hubs.tfvars

```diff
@@ -19,11 +19,11 @@ enable_filestore      = true
 filestore_capacity_gb = 5120
 
 notebook_nodes = {
-  # FIXME: Rename this to "n2-highmem-4" when given the chance and no such nodes are running
-  "user" : {
+  "n2-highmem-4" : {
     min : 0,
     max : 100,
     machine_type : "n2-highmem-4",
+    node_version : "1.27.4-gke.900",
```

Comment:
Is this pin needed if it's the same with the  `notebook_nodes_version : "1.27.4-gke.900", "1.27.4-gke.900"`?

Reply:
Nice catch, nope - removed it in a dedicated commit